src/service/cron.js

```diff
@@ -3,10 +3,10 @@ var CronJob = require("cron").CronJob;
 
 export class CronController {
     cron(email) {
-        // new CronJob("*/15 * * * *", function() {
-            inbox.fetch_email(email); // running this function every 15 min
-            // inbox.Before_date_email(email, new Date());
-        // }, null, true);
+        new CronJob("*/15 * * * *", function() {
+            inbox.fetchEmail(email); // running this function every 15 min
+            inbox.beforeDateEmail(email, new Date());
```

Comment:
this is wrong. here you are passing new Date() everytime.

how will this function work. 

this will always fetch the same again and again. we need to go back in dates, not ahead